samples/trigger_joins.md

```diff
@@ -17,310 +16,326 @@ def get_request_string(use_trigger_1=True, use_trigger_2=True):
-                [0;32m"join_node.outputs"[0m[1;39m
-              [1;39m][0m[1;39m
-            [1;39m}[0m[1;39m,
-            [0m[34;1m"kubernetesMeta"[0m[1;39m: [0m[1;39m{}[0m[1;39m
-          [1;39m}[0m[1;39m,
-          [0m[34;1m"state"[0m[1;39m: [0m[1;39m{
-            [0m[34;1m"pipelineVersion"[0m[1;39m: [0m[0;39m1[0m[1;39m,
-            [0m[34;1m"status"[0m[1;39m: [0m[0;32m"PipelineReady"[0m[1;39m,
-            [0m[34;1m"reason"[0m[1;39m: [0m[0;32m"created pipeline"[0m[1;39m,
-            [0m[34;1m"lastChangeTimestamp"[0m[1;39m: [0m[0;32m"2022-11-16T19:29:58.778539435Z"[0m[1;39m
-          [1;39m}[0m[1;39m
-        [1;39m}[0m[1;39m
-      [1;39m][0m[1;39m
-    [1;39m}[0m
+```
 
+```bash
+seldon pipeline status triggers_join_inputs -w PipelineReady | jq .
+```
 
+```json
+{
+  "pipelineName": "triggers_join_inputs",
+  "versions": [
+    {
+      "pipeline": {
+        "name": "triggers_join_inputs",
+        "uid": "cdqjkdhqa12c739ab3t0",
+        "version": 1,
+        "steps": [
+          {
+            "name": "join_node",
+            "inputs": [
+              "triggers_join_inputs.inputs.INPUT1",
+              "triggers_join_inputs.inputs.INPUT2"
+            ],
+            "triggers": [
+              "triggers_join_inputs.inputs.TRIGGER1",
+              "triggers_join_inputs.inputs.TRIGGER2"
+            ],
+            "triggersJoin": "ANY"
+          }
+        ],
+        "output": {
+          "steps": [
+            "join_node.outputs"
+          ]
+        },
+        "kubernetesMeta": {}
+      },
+      "state": {
+        "pipelineVersion": 1,
+        "status": "PipelineReady",
+        "reason": "created pipeline",
+        "lastChangeTimestamp": "2022-11-16T19:29:58.778539435Z"
+      }
+    }
+  ]
+}
 
-```python
-!cat ./pipelines/triggers_join_inputs.yaml
 ```
 
-    apiVersion: mlops.seldon.io/v1alpha1
-    kind: Pipeline
-    metadata:
-      name: triggers_join_inputs
-    spec:
-      steps:
-        - name: join_node
-          inputs:
-            - triggers_join_inputs.inputs.INPUT1
-            - triggers_join_inputs.inputs.INPUT2
-          triggers:
-            - triggers_join_inputs.inputs.TRIGGER1
-            - triggers_join_inputs.inputs.TRIGGER2
-          triggersJoinType: any
-      output:
-        steps:
-          - join_node
+```bash
+cat ./pipelines/triggers_join_inputs.yaml
+```
 
+```yaml
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: triggers_join_inputs
+spec:
+  steps:
+    - name: join_node
+      inputs:
+        - triggers_join_inputs.inputs.INPUT1
+        - triggers_join_inputs.inputs.INPUT2
+      triggers:
+        - triggers_join_inputs.inputs.TRIGGER1
+        - triggers_join_inputs.inputs.TRIGGER2
+      triggersJoinType: any
+  output:
+    steps:
+      - join_node
 
+```
 
-```python
+```bash
 request_string = get_request_string(use_trigger_1=True, use_trigger_2=True)
```

Comment:
:information_source: This is a case of the shell script inference getting it wrong (this isn't a _pure_ bash block), but equally it's not a pure Python block, so we couldn't simply copy/paste anyway without IPython or Jupyter.  If using those then it's probably best to just use the notebooks directly as they're executable.

All in all, I think this is a reasonable mistake to let slip.